packages/messaging-system/config/environment.js

```diff
@@ -0,0 +1,16 @@
+// External dependencies
+const { join } = require('path');
+
+// Internal dependencies
+const { supportedEnvs } = require('../src/utils/constants');
+
+let path;
+const filename = '.env';
+
+if (process.env.NODE_ENV === supportedEnvs.PRODUCTION) {
+  path = `/src/${filename}.${process.env.NODE_ENV}`;
+} else {
+  path = `${join(__dirname, '../../..')}/${filename}`;
+}
+
+module.exports = path;
```

Comment:
We don't need this, there already is a configuration for this here: https://github.com/Platzi-Master-C9/booking-ui/blob/main/dotenv.config.js

Reply:
Well there is dotenv configuration, but we can't use that because we are exporting nothing, so next.js doesn't run that file. I could refactor it to put that file in shared package and export the path.

Reply:
Forget about it, I have to correct the way shared package was used . 
Here I have two options.
- Let that file like that
- refactor dotenv.config.js to export path and use it when we are using env vars, but it will break our architecture